test/intl402/DurationFormat/instance/length.js

```diff
@@ -27,6 +27,8 @@ includes: [propertyHelper.js]
 
 assert.sameValue(Intl.DurationFormat.length, 0);
 
-verifyNotEnumerable(Intl.DurationFormat, "length");
-verifyNotWritable(Intl.DurationFormat, "length");
-verifyConfigurable(Intl.DurationFormat, "length");
+verifyProperty(Intl.DurationFormat, 'length', {
+  enumerable: false,
```

Comment:
Maybe:
```suggestion
verifyProperty(Intl.DurationFormat, 'length', {
  value: 0,
  enumerable: false,
```

Reply:
Thanks , @Ms2ger also deleted the previous assertion for value